addons/base_import/static/src/import_action/import_action.js

```diff
@@ -93,6 +93,10 @@ export class ImportAction extends Component {
         return this.model.importOptions;
     }
 
+    get webOptions() {
```

Comment:
I would remove this getter as used only once in the template, and value is easy to get